+brillem/lattice.m

```diff
@@ -14,14 +14,21 @@
 % You should have received a copy of the GNU General Public License
 % along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-function [dlat,rlat] = lattice(lens, angs, rord, varargin)
-if nargin < 3 || isempty(rord)
+function [dlat,rlat] = lattice(lens, angs, raddeg, rord, varargin)
+if nargin < 4 || isempty(rord)
     rord = 'direct';
 end
+if nargin < 3 || isempty(raddeg)
+    raddeg = 'degrees';
+end
 if nargin < 2 || isempty(angs)
     angs = [90,90,90];
 end
 
+if strncmp(raddeg, 'rad', 3)
```

Comment:
The `brille::Lattice` classes _should_ be able to distinguish between radian and degree angles automatically by comparing the three provided angles to 2π (if all are less than 2π they are _almost certainly_ in radian, otherwise they should be degrees).
Did you find a case where the C++ class failed to identify radian (or degree) angles?

Reply:
Ah no... I changed the Matlab code without first running it. I think the original Matlab code had a `degrees` argument or something... but the call to it did not have (or the other way around).